testsuite/features/support/commonlib.rb

```diff
@@ -66,6 +66,21 @@ def product
   raise 'Could not determine product'
 end
 
+def product_version
```

Comment:
In theory, it should not collide, but we might need to rename this parameter here:

``` 
When(/^I (deselect|select) "([^\"]*)" as a (SUSE Manager|Uyuni) product$/) do |select, product, product_version|
  if $product == product_version
    step %(I #{select} "#{product}" as a product)
  end
end
```

Reply:
This is a very nice to have function, thank you